### PR TITLE
AO3-6405 Task for removing multiple ratings on a work

### PR DESCRIPTION
**Pull request by ellieyhcheng**
# Pull Request Checklist

* [x] Have you read ["How to write the perfect pull request"](https://github.blog/2015-01-21-how-to-write-the-perfect-pull-request/)?
* [x] Have you read the [contributing guidelines](https://github.com/otwcode/otwarchive/blob/master/CONTRIBUTING.md)?
* [x] Have you added [tests for any changed functionality](https://github.com/otwcode/otwarchive/wiki/Automated-Testing)?
* [x] Have you added the [Jira](https://otwarchive.atlassian.net) issue number
  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
* [x] Do you have fewer than 5 pull requests already open? If not, please wait
  until they are reviewed and merged before creating new pull requests.

## Issue
https://otwarchive.atlassian.net/browse/AO3-6405

## Purpose
Adds task for removing multiple ratings on a work, which was [a bug fixed about a year ago](https://otwarchive.atlassian.net/browse/AO3-6533) (sorry for the wait). 

## Testing Instructions
`docker compose run --rm test bundle exec rspec spec/lib/tasks/after_tasks.rake_spec.rb` passes. 

## References
N/A

## Credit
ellieyhcheng (she/her)

### CODE REVIEW

**Review by brianjaustin — CHANGES_REQUESTED** [review]
Thanks! A couple suggestions, plus the Rubocop ones. Please let me know if anything I said needs additional clarification!

File: lib/tasks/after_tasks.rake [hunk 1]
```diff
@@ -67,6 +67,39 @@ namespace :After do
...
   end
 
+  desc "Clean up multiple rating tags"
+  task(clean_up_multiple_ratings: :environment) do
+    default_rating_tag = Rating.find_by!(name: ArchiveConfig.RATING_DEFAULT_TAG_NAME)
+    invalid_works = Work.select { |work| work.ratings.size > 1 }
+    puts "There are #{invalid_works.size} works with multiple ratings."
+
+    work_ids = []
+    invalid_work_ids = []
```
[the remark this answers is not in the document]
Comment (ellieyhcheng) [reply lib/tasks/after_tasks.rake:77]: Good idea, will do!

File: lib/tasks/after_tasks.rake [hunk 2]
```diff
@@ -67,6 +67,39 @@ namespace :After do
     end
   end
 
+  desc "Clean up multiple rating tags"
+  task(clean_up_multiple_ratings: :environment) do
+    default_rating_tag = Rating.find_by!(name: ArchiveConfig.RATING_DEFAULT_TAG_NAME)
+    invalid_works = Work.select { |work| work.ratings.size > 1 }
```
[the remark this answers is not in the document]
Comment (ellieyhcheng) [reply lib/tasks/after_tasks.rake:73]: I'm not familiar with Elasticsearch, so I'm having some trouble integrating this with the spec tests. I can't quite figure out how to fill the indexer for the test (the query returns 0 works in the spec test). Is there a pointer to another spec test that does so?

[the remark this answers is not in the document]
Comment (ellieyhcheng) [reply lib/tasks/after_tasks.rake:73]: Nevermind! The thing I tried after writing this comment worked (of course, it does). Will update the PR soon.

**Review by brianjaustin — APPROVED** [review]
Thanks!

**Pull request merged**